roles/rabbitmq/tasks/main.yml

```diff
@@ -15,3 +15,13 @@
     enabled: yes
     state: started
   tags: rabbitmq
+
+- name: Add RabbitMQ plugins
+  become: yes
+  rabbitmq_plugin:
+    name: "{{ item }}"
```

Comment:
From http://docs.ansible.com/ansible/latest/rabbitmq_plugin_module.html I see the `names` (not `name`) is comma-separated and there is additional `new_only` option which as I understand might disable previously enabled plugins.

So this logic needs to be double-checked.

Reply:
ok. `name` is an alias for `names`, but I guess it would make sense to use the csv list instead of using an ansible with loop. I didn't see `new_only`: I think that should definitely be on by default.